action.yml

```diff
@@ -1,10 +1,10 @@
 name: 'Checkout'
-description: 'Checkout a Git repository.'
+description: 'Clone and checkout a Git repository at a particular version'
 inputs: 
   repository:
     description: 'Repository name'
   ref:
-    description: 'Ref to checkout (SHA, branch, tag)'
+    description: 'Reference or ID to checkout (SHA, branch, tag)'

```

Comment:
`The branch, SHA or tag to checkout` might be a little less verbose.

Reply:
either way seems good. Reference and ID aligns with git terminology, and the name of the input

Reply:
If users can specify just a branch name here (eg, `master`) then I think that saying "Branch" up-front is ok.  But if it needs to be in proper reference format (eg, `refs/heads/master`) then I think that we should say "Reference or ID" up front.

Reply:
`master` and `refs/heads/master` are both supported

Reply:
Nice.  In that case I agree with @thboop.  Will push an update.